### DISCUSSION TITLE
[Feature] getAssetDuplicates to include similarity/"distance" index

### DISCUSSION THREAD

**Discussion by Wilm0r** [immich-app/immich#13753] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [x] Yes

### The feature

I love duplicate detection and how it also finds reasonably similar images that can be stacked. But I'd love if I could differentiate between those and outright dupes automatically as I have a lot of those during the process of importing from Google Photos, my current phone and older ones, etc. And especially Nikon SnapBridge causes a lot of lower-resolution dupes of larger original photos from the camera directly.

Would be cool if the `getAssetDuplicates` API call could include the similarity index as returned by `searchDuplicates`? (Though interesting question is, which exact delta to include when there are more than 2 pictures in the dupes set?)

### Platform

- [X] Server
- [ ] Web
- [ ] Mobile

**Comment by mertalev** [comment]
The distance isn't stored anywhere, so this would require a different (and probably more complex / less efficient) data model and clustering algorithm. But you can set the duplicate detection threshold in the machine learning settings to something lower and re-run Duplicate Detection on all assets in the job panel.
